### ISSUE TITLE
MX vertical cannot be repsent correctly and cannot be reassigned for the dpi switch button

### ISSUE THREAD

**Issue by wnykuang** [libratbag/piper#869]
**Information**

- `ratbagd` version (`0.16`):
- Piper version:  Version: 0.6
- Distribution: Kubuntu
- Device:  MX Vertical
- Kernel version (ex. `uname -srmo`): `Linux 5.19.0-38-generic x86_64 GNU/Linux`

**Describe the bug**
- MX vertical's dpi switch buttom cannot be recoginized correctly and cannot be assigned to other key in piper GUI.

**To Reproduce**
Insert the MX vertical and open piper will find the dpi button matched to backward.

**Additional context**
I believe that's because MX vertical's dpi button's value is different from ordinary ones.
I used logidops to see the dpi key's value is `0xfd` which full table is:

```
[DEBUG] /dev/hidraw3:2 remappable buttons:
[DEBUG] CID  | reprog? | fn key? | mouse key? | gesture support?
[DEBUG] 0x50 |         |         | YES        | 
[DEBUG] 0x51 |         |         | YES        | 
[DEBUG] 0x52 | YES     |         | YES        | YES
[DEBUG] 0x53 | YES     |         | YES        | YES
[DEBUG] 0x56 | YES     |         | YES        | YES
[DEBUG] 0xd7 | YES     |         |            | YES
[DEBUG] 0xfd | YES     |         | YES        | YES
```
And it could be mismatched since 0xd7 is other mouse's backward key.

**Comment by staticssleever668** [comment]
I don't really understand what are you trying to say. Is the DPI switch button not shown in Piper? (a screenshot would help).
Is it shown in ratbagctl? (I think you can check it like this: `ratbagctl Logitech info`).

**Comment by wnykuang** [comment]
```
ratbagctl Logitech info
hollering-marmot - Logitech MX Vertical
             Model: usb:046d:407b:0
 Number of Buttons: 7
    Number of Leds: 0
Number of Profiles: 1
Profile 0: (active)
  Name: n/a
  Resolutions:
    0: 1500dpi (active)
  Button: 0 is mapped to 'button 1'
  Button: 1 is mapped to 'button 2'
  Button: 2 is mapped to 'button 3'
  Button: 3 is mapped to 'button 4'
  Button: 4 is mapped to 'button 5'
  Button: 5 is mapped to UNKNOWN
  Button: 6 is mapped to UNKNOWN

```

Sorry, the screenshot is missed since I forgot to upload.  :'( 
![Screenshot_20230412_155427](https://user-images.githubusercontent.com/3423210/231603721-843275bf-8a13-4223-ba82-190ece0be764.png)

You could notice, the backward key is mapping to dpi swtich key (the white small one) and the true backward button has been maked as unknown.

I believe there is something wrong in the configuration file. Could you give a pointer? I could fix it. (I have set my MX Vertical up properly with logidops so I am sure this thing is doable.)

**Comment by wnykuang** [comment]
BTW, Thanks for such fast response. :D

**Comment by staticssleever668** [comment]
This has been fixed by #713 (available in Piper v0.7).
